Libraries/NativeAnimation/Nodes/RCTPropsAnimatedNode.m

```diff
@@ -33,6 +45,17 @@ - (void)connectToView:(NSNumber *)viewTag
 
 - (void)disconnectFromView:(NSNumber *)viewTag
 {
+  // Restore the default value for all props that were modified by this node.
+  for (NSString *key in [_propsDictionary allKeys]) {
```

Comment:
`allKeys` is documented as a property, write `_propsDictionary.allKeys`: https://developer.apple.com/reference/foundation/nsdictionary/1409150-allkeys?language=objc